BasicCalculator.js

```diff
@@ -0,0 +1,407 @@
+function add(A, B){
```

Comment:
Soooo, fun fact...you committed your node_modules folder! To avoid doing this, specify which files you want to commit using `git add`.